src/site/xdoc/server/config-spring-jpa-postgres.xml

```diff
@@ -0,0 +1,89 @@
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.    
+-->
+<document>
+
+ <properties>
+  <title>Apache James Server 3 - Spring JPA With Postgres Configuration</title>
+ </properties>
+
+<body>
+
+
+    <section name="Introduction">
+      <p>Consult <a href="http://james.apache.org/server/config-system.html">System Configuration</a> to get template, some examples and hints for the configuration with JPA.</p>
+    </section>
+
+    <section name="Current Supported Sequence Database">
+      <p>James supports DERBY as default database, it does not need to do any configuration in case of using Derby</p>
+    </section>
+
+    <section name="Spring JPA With Postgres Configuration">
+      <p>In case of using POSTGRES as sequence database, please following those steps for configuration</p>
+      <subsection name="Step 1: Create database on Postgres">
+
+        <p>It needs to install Postgres or run Postgres with Docker</p>
+        <p>Postgres does not create database on the fly so it needs to create database by manually. The command should be like this</p>
+
+        <source>psql -U postgres -tc "SELECT 1 FROM pg_database WHERE datname = '<strong><italic>database_name</italic></strong>'" | grep -q 1 || psql -U postgres -c "CREATE DATABASE <strong><italic>database_name</italic></strong>"</source>
+
+        <dl>
+          <dt><strong>database_name</strong></dt>
+          <dd>The name of database</dd>
+        </dl>
+
+      </subsection>
+
+      <subsection name="Step 2: Point to the database n in configuration file">
+        <p>Change the content of <italic>james-database.properties</italic> like this. Point to the 'database_name' as above step</p>
+        <source>
+          database.driverClassName=org.postgresql.Driver
+          database.url=jdbc:postgresql://<strong>ip_of_postgres</strong>/<strong>database_name</strong>
+          database.username=postgres
+          database.password=postgres
+
+          vendorAdapter.database=POSTGRESQL
+        </source>
+        <p>In case of using docker image of spring james. Create and edit james-database.properties file on local with above content then copy to docker container using</p>
+
+        <source>
+          docker cp <italic>path_to_james_database_properties_file_on_local</italic> <italic>james_container_id</italic>:/root/james-server-app-3.0.0-RC2/conf
+        </source>
+      </subsection>
+
+      <subsection name="Step 3: Copy driver class to right place">
+        <p>Following this link <a href="https://jdbc.postgresql.org/download.html">Postgres Driver</a> to download the right version of driver class </p>
+        <p>Copy that jar file to the lib folder of Spring James</p>
+        <p>In case of using docker image of spring james. Download jar file to local then copy to docker container using</p>
```

Comment:
Download the jar then make it available in the classpath of your James instance using an additional docker volume: